### DISCUSSION TITLE
2024-geeknews-meetup-summary/

### DISCUSSION THREAD

**Discussion by giscus** [JungWinter/blog#46] [Comments]
# 2024-geeknews-meetup-summary/

얼마 전 12월 12일에 "라이프 해킹 - 즐거운 삶을 위하여"라는 주제로 처음 열리는 긱뉴스 밋업에 다녀왔다.
매일 즐겨보던 긱뉴스기도 하고 라이트닝 토크에 재미있어 보이는 주제도 많아 꼭 가고 싶었는데, 평소 활발히는 아니어도 종종 글도 올리고 …

https://www.winterjung.dev/2024-geeknews-meetup-summary/

**Comment by xguru** [comment]
우와 감사합니다. 제가 후기를 작성해보려고 했는데 이 글을 긱뉴스에 포스팅하는 것으로 대신하겠습니다 ;)

**Comment by seonWKim** [comment]
우와.. 평소에 뵙고 싶었던 분들이 많이 계시네요. 기회가 된다면 나중에 참석해보고싶네요 :)

**Comment by Edunga1** [comment]
후기 감사합니다. 저는 당첨되었는데, 하필 당일 몸살이 나서 못가서 너무 아쉬워요.

**Comment by jaewon-kim** [comment]
신청했으나 안되어 매우 궁금했는데 잘 정리해주신 글 보고 궁금증이 어느정도 해소 되었습니다. 
잘 보고 갑니다. 감사합니다.

**Comment by billygoo** [comment]
발표 정리 내용을 읽다보니 참석해보고 싶은 마음이 솟구치게 하는 글이네요. 정리 감사합니다. ^^

**Comment by inchanS** [comment]
참석하지 못해 못내 아쉬웠는데, 후기 감사합니다.
